keyboards/pimoroni/keybow2040/keybow2040.c

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "rgb_matrix.h"
+
+void keyboard_post_init_user(void) {
+	debug_enable = true;
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+#ifdef CONSOLE_ENABLE
+	dprintf("KL: kc: 0x%04X, col: %2u, row: %2u, "
+		"pressed: %u, time: %5u, "
+		"int: %u, count: %u\n",
+		keycode, record->event.key.col, record->event.key.row,
+		record->event.pressed, record->event.time,
+		record->tap.interrupted, record->tap.count);
+#endif
+  return true;
+}
+
+// https://github.com/adafruit/Adafruit_CircuitPython_IS31FL3731/blob/main/adafruit_is31fl3731/keybow2040.py
+
+const is31_led PROGMEM g_is31_leds[RGB_MATRIX_LED_COUNT] = {
+	{ 0, 120, 88, 104 }, // 0, 0
+	{ 0, 136, 40, 72 },  // 1, 0
+	{ 0, 112, 80, 96 },  // 2, 0
+	{ 0, 128, 32, 64 },  // 3, 0
+	{ 0, 121, 89, 105 }, // 0, 1
+	{ 0, 137, 41, 73 },  // 1, 1
+	{ 0, 113, 81, 97 },  // 2, 1
+	{ 0, 129, 33, 65 },  // 3, 1
+	{ 0, 122, 90, 106 }, // 0, 2
+	{ 0, 138, 25, 74 },  // 1, 2
+	{ 0, 114, 82, 98 },  // 2, 2
+	{ 0, 130, 17, 66 },  // 3, 2
+	{ 0, 123, 91, 107 }, // 0, 3
+	{ 0, 139, 26, 75 },  // 1, 3
+	{ 0, 115, 83, 99 },  // 2, 3
+	{ 0, 131, 18, 67 },  // 3, 3
+};
+
+led_config_t g_led_config = {
+	// Matrix location to LED index
+	{
+		{ 0x0, 0x1, 0x2, 0x3, },
+		{ 0x4, 0x5, 0x6, 0x7, },
+		{ 0x8, 0x9, 0xA, 0xB, },
+		{ 0xC, 0xD, 0xE, 0xF, },
+		{ NO_LED, NO_LED, NO_LED, NO_LED, },
+	},
+
+	// LED index to physical location
+	// equally spaced horizontally;
+	// vertically LEDs are 5mm from the top edge
+	{
+		{25,7}, {75,7}, {125,7}, {175,7},
+		{25,57}, {75,57}, {125,57}, {175,57},
+		{25,107}, {75,107}, {125,107}, {175,107},
+		{25,157}, {75,157}, {125,157}, {175,157},
+	},
+
+	// LED index to flags
+	{
+		LED_FLAG_KEYLIGHT, LED_FLAG_KEYLIGHT,
+		LED_FLAG_KEYLIGHT, LED_FLAG_KEYLIGHT,
+		LED_FLAG_KEYLIGHT, LED_FLAG_KEYLIGHT,
+		LED_FLAG_KEYLIGHT, LED_FLAG_KEYLIGHT,
+		LED_FLAG_KEYLIGHT, LED_FLAG_KEYLIGHT,
+		LED_FLAG_KEYLIGHT, LED_FLAG_KEYLIGHT,
+		LED_FLAG_KEYLIGHT, LED_FLAG_KEYLIGHT,
+		LED_FLAG_KEYLIGHT, LED_FLAG_KEYLIGHT,
+	}
+};
```

Comment:
The current preference is to convert this data into `info.json`.
`qmk info -f json -kb pimoroni/keybow2040` should be able to make the JSON data for `rgb_matrix` from this.

Reply:
`qmk migrate -kb pimoroni/keybow2040` may be simpler, as it should move most of the settings it can into the json and remove the legacy config.

Reply:
Migrated to info.json in round two

@drashna I tried `qmk migrate` but that only migrated the `center_point`. `qmk info -f json` was able to turn the `g_led_config` into JSON `rgb_matrix` settings, so I used that. I was surprised that `qmk info -f json` knows more than `qmk migrate`